tgui/packages/tgui/interfaces/SecurityRecords/constants.tsx

```diff
@@ -0,0 +1,8 @@
+export const CRIMESTATUS2COLOR = {
```

Comment:
would it be possible to use scss rather than hard coded strings?

Reply:
I can't find anywhere in the code an example of where this has been done before. Are you sure it's possible / better than just using the string like all other components?